prototype/src/js/worlds/office.js

```diff
@@ -0,0 +1,1198 @@
+                    "In astronomy and astrobiology, the circumstellar habitable zone (CHZ), or simply the habitable zone, is the range of orbits around a star within which a planetary surface can support liquid water given sufficient atmospheric pressure.[1][2][3][4][5] The bounds of the CHZ are based on Earth's position in the Solar System and the amount of radiant energy it receives from the Sun. Due to the importance of liquid water to Earth's biosphere, the nature of the CHZ and the objects within it may be instrumental in determining the scope and distribution of Earth-like extraterrestrial life and intelligence."
+            }
+        ]
+    },
+    {
+        name: 'Demilitarized Zone',
+        desc:
+            'A demilitarized zone, DMZ or DZ[1] is an area in which treaties or agreements between nations, military powers or contending groups forbid military installations, activities or personnel.',
+        lat: 39.961215,
+        lng: -75.15427,
+        bearing: 275,
+        tour: [
+            {
+                image: 'demilitarized.jpg',
+                caption:
+                    'Many demilitarized zones are considered neutral territory because neither side is allowed to control it, even for non-combat administration. Some zones remain demilitarized after an agreement has awarded control to a state which (under the DMZ terms) had originally ceded its right to maintain military forces in the disputed territory. It is also possible for powers to agree on the demilitarization of a zone without formally settling their respective territorial claims, enabling the dispute to be resolved by peaceful means such as diplomatic dialogue or an international court.'
+            }
+        ]
+    },
+    {
+        name: 'Eastern Time Zone',
+        desc:
+            'The Eastern Time Zone (ET) encompasses 17 U.S. states in the eastern part of the contiguous United States, parts of eastern Canada, the state of Quintana Roo in Mexico, Panama in Central America, and the Caribbean Islands.',
+        lat: 39.96133,
+        lng: -75.15393,
+        bearing: 95,
+        tour: [
+            {
+                image: 'easterntime.jpg',
+                caption:
+                    'In the northern parts of the time zone, on the second Sunday in March, at 2:00 a.m. EST, clocks are advanced to 3:00 a.m. EDT leaving a one-hour "gap". On the first Sunday in November, at 2:00 a.m. EDT, clocks are moved back to 1:00 a.m. EST, thus "duplicating" one hour. Southern parts of the zone (Panama and the Caribbean) do not observe daylight saving time.'
+            }
+        ]
+    }
+];
+
+export function styleArea(feature, layer) {
+    function getAreaColor(areaType) {
+        // colors from http://www.colourlovers.com/palette/1473/Ocean_Five
+        // actually from http://www.colourlovers.com/palette/932683/Compatible
+        let color;
+        switch (areaType) {
+            case 'Meeting':
+                color = '#8EBE94';
```

Comment:
Nitpicky but this function could also return in each of these case statements instead of assigning and then `break`ing.

Reply:
I've always had a resistance to multiple return statements, but perhaps it's time to let that go. Will keep in mind.